### PR TITLE
Doischer/2491 implement search for scripts in mapping

### PR DESCRIPTION
**Pull request by tomdoischer**
https://redmine.czechidm.com/issues/2491

### CODE REVIEW

File: Realization/frontend/czechidm-acc/src/services/SystemAttributeMappingService.js [hunk 1]
```diff
@@ -25,4 +25,18 @@ export default class SystemAttributeMappingService extends Services.AbstractServ
   getDefaultSearchParameters() {
     return super.getDefaultSearchParameters().setName(Domain.SearchParameters.NAME_QUICK).clearSort().setSort('name');
   }
+
+  getScriptUsage(code) {
+    return Services.RestApiService
+    .get(`${ this.getApiPath() }/script-usage/${ encodeURIComponent(code) }`)
+    // .get(`/script-usage/${ encodeURIComponent(code) }`)
```
Comment (kucera-roman) [inline Realization/frontend/czechidm-acc/src/services/SystemAttributeMappingService.js:32]: remove commented code please.

**Review by kucera-roman — CHANGES_REQUESTED** [review]
LGTM, working nicely. Fix just one small thing.

File: Realization/frontend/czechidm-acc/src/services/SystemAttributeMappingService.js (hunk 1 above)
Comment (tomdoischer) [reply Realization/frontend/czechidm-acc/src/services/SystemAttributeMappingService.js]: Thanks, done. :)

**Review by kucera-roman — APPROVED** [review]
LGTM

**Pull request merged**